sdk/eventhub/azure-eventhub/setup.py

```diff
@@ -55,7 +55,7 @@
     author_email='azpysdkhelp@microsoft.com',
     url='https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/eventhub/azure-eventhub',
```

Comment:
Do we need to update the URL to point to the branch? Or will the change the URL for the GA EH release as well...?

Reply:
I remember the doc guideline is to point to the main branch links even within a feature branch.